Galaxus.Functional/(Option)/(Extensions)/OptionLinqExtensions.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Galaxus.Functional
```

Comment:
Since all of these methods are supposed to be alternatives to System.Linq, what about putting them into a `Galaxus.Functional.Linq` namespace?